### ISSUE TITLE
Fix property transmission in a mixed Python/Concert environment

### ISSUE THREAD

**Issue by matze** [ufo-kit/uca-net#9]
So, apparently there are a few problems with the Concert integration. The telephone is not a good way to track these issues here, so please comment here @tfarago.

Passing strings as enum values works from `uca-grab`

```bash
$shell1 ucad mock
$shell2 uca-grab -p "trigger-source=UCA_CAMERA_TRIGGER_SOURCE_EXTERNAL" net
```

and in Concert

```python
from concert.devices.cameras.uca import Camera

camera = Camera('net')
camera.trigger_source = camera.trigger_sources.EXTERNAL
```

but as far as I understood not so well with custom enum values @tfarago ?

**Comment by matze** [comment]
Custom enum types and values work the same:

```bash
$shell1 ucad mock
$shell2 uca-grab -p "test-enum=UCA_MOCK_CAMERA_TEST_ENUM_FOO" net
```

```python
from concert.devices.cameras.uca import Camera

camera = Camera('net')
camera.test_enum = camera.uca.enum_values.test_enum.UCA_MOCK_CAMERA_TEST_ENUM_FOO
```

**Comment by matze** [comment]
@tfarago: could you provide me an example that is not working?

**Comment by tfarago** [comment]
```python
tomography > camera.acquire_mode = camera.uca.enum_values.acquire_mode.
camera.uca.enum_values.acquire_mode.UCA_PCO_CAMERA_ACQUIRE_MODE_AUTOUCA_PCO_CAMERA_ACQUIRE_MODE_EXTE�����^?
camera.uca.enum_values.acquire_mode.UCA_PCO_CAMERA_ACQUIRE_MODE_EXTE�����^?

tomography > camera.acquire_mode = camera.uca.enum_values.acquire_mode.UCA_PCO_CAMERA_ACQUIRE_MODE_
```
On the server side I also see in output some weird hexa characters.

**Comment by tfarago** [comment]
Is this enough?

**Comment by matze** [comment]
Can you stop the daemon?

**Comment by tfarago** [comment]
done

**Comment by matze** [comment]
Should be fixed with e16b1811d62b7493133093205d498a30b6f04dae. Re-open if necessary.
